Source/inv.h

```diff
@@ -78,54 +78,80 @@ extern bool invflag;
 extern bool drawsbarflag;
 extern const InvXY InvRect[73];
 
-void FreeInvGFX();
-void InitInv();
-
-/**
- * @brief Render the inventory panel to the given buffer.
- */
-void DrawInv(CelOutputBuffer out);
-
-void DrawInvBelt(CelOutputBuffer out);
-bool AutoEquipEnabled(const PlayerStruct &player, const ItemStruct &item);
-bool AutoEquip(int playerNumber, const ItemStruct &item, bool persistItem = true);
-bool AutoPlaceItemInInventory(int playerNumber, const ItemStruct &item, bool persistItem = false);
-bool AutoPlaceItemInInventorySlot(int playerNumber, int slotIndex, const ItemStruct &item, bool persistItem);
-bool AutoPlaceItemInBelt(int playerNumber, const ItemStruct &item, bool persistItem = false);
-bool GoldAutoPlace(int pnum);
-void CheckInvSwap(int pnum, BYTE bLoc, int idx, WORD wCI, int seed, bool bId, uint32_t dwBuff);
-void inv_update_rem_item(int pnum, BYTE iv);
+class Inventory {
+
+private:
+	bool AutoEquip(int playerNumber, const ItemStruct &item, inv_body_loc bodyLocation, bool persistItem);
+	bool CanEquip(const ItemStruct &item);
+	bool CanEquip(int playerNumber, const ItemStruct &item, inv_body_loc bodyLocation);
+	bool CanWield(int playerNumber, const ItemStruct &item);
+	bool CanBePlacedOnBelt(const ItemStruct &item);
+	void CheckBookLevel(int pnum);
+	void CheckInvCut(int pnum, int mx, int my, bool automaticMove);
+	void CheckInvPaste(int pnum, int mx, int my);
+	void CheckQuestItem(int pnum);
+	void CleanupItems(ItemStruct *item, int ii);
+	bool FitsInBeltSlot(const ItemStruct &item);
+	int SwapItem(ItemStruct *a, ItemStruct *b);
+	bool WeaponAutoPlace(int pnum);
+
+public:
+	void FreeInvGFX();

```

Comment:
Let's remove `Inv` and `inv_` in method names?

Reply:
Thanks for the suggestion, I only kept `UseInvItem` because it clashes with `items#UseItem`